internal/pkg/dl/policies.go

```diff
@@ -79,3 +90,18 @@ func CreatePolicy(ctx context.Context, bulker bulk.Bulk, policy model.Policy, op
 	}
 	return bulker.Create(ctx, o.indexName, "", data)
 }
+
+// FindPolicyByID find policy by ID
+func FindPolicyByID(ctx context.Context, bulker bulk.Bulk, policyID string) (policy model.Policy, err error) {
```

Comment:
The `policy.Monitor` keeps the latest revision of each policy in memory. It would be better to fetch the policy from there instead of going to elasticsearch, as fleet-server already has it cached in memory.

Reply:
Sounds good, was not aware, will update.